src/fri/naive.md

```diff
@@ -0,0 +1,157 @@
+
+With the same folding setup, the final polynomial will be always the same value. Verifier can check against the same final value against the reduced values from different evaluation points.
+
+If the final polynomial can be derived after expected rounds of folding, it implies the $p_0$ has an expected upper bound degree. In other words, if the final $p_k$ can be derived after $k=log(d)$ rounds of folding from $p_0$ which is of degree $d$, then $p_0$ has upper bound degree $d$.
+
+More formally, there are two properties:
+- the number of rounds $k=log(d)$ to derive the final polynomial implies the expected upper bound degree $d$
+- the same final value can be re-used to check consistency of the evaluation claims across layers
+
+We will refer this folding example again with some actual evaluations to demonstrate the interesting relationship among them.
+
+## What is the problem?
+
+Using the notation from [what is proving](https://nmohnblatt.github.io/zk-jargon-decoder/intro_to_zk/what_is_proving.html), we define the following:
+
+$$
+\mathcal{L}(\mathcal{R}_{\text{consistent-layers}})
+$$
+
+for instance, $\left(a_1, w_1\right) \in \mathcal{R}$, where $a_1$ can represent a FRI query and constant value from $p_2$, while $w_1$ represents the corresponding evaluations across layers.
+
+So the problem is to do the check to see if $a_1$ is in $\mathcal{L}$. In other words, the verifier checks if the prover can provide $w_1$ satisfying the query $a_1$.
+
+## naive FRI
+
+Let's assume prover is honest guy in this naive version of FRI. The goal is for verifier to do fast low degree test using the proof from prover. 
+
+### Hard honest work by prover
+
+Prover can use the folded polynomial to obtain the evaluations for each layers. These evaluations of layers can serve as a proof for the verifier to sample for their consistency across layers.
+
+Using the same polynomial example shown earlier, here are two simple calculation examples.
+
+For evaluation point at *1*:
+
+$$
+\begin{align} 
+\text{Layer 1:} & \\
+x_0&=1 \\
+p_0(x_0) &= 1 + 2x_0 + 3x_0^2 \\ 
+p_0(1) &= 6 \\ 
+p_0(-1) &= 2 \\\\
+\text{Layer 2:} & \\
+x_1&=x_0^2 \\
+p_1(x_1) &= 3 + 3x_1 \\
+p_1(1^2) &= 6 \\ 
+p_1(-1^2) &= 0 \\\\
+\text{Layer 3:} & \\
+x_2&=x_1^2 \\
+p_2(x_2) &= 6 \\
+p_2((1^2)^2) &= 6 \\\\
+\end{align}
+$$
+
+Similarly, for evaluation point at *2*:
+
+$$
+\begin{align} 
+\text{Layer 1:} & \\
+x_0&=2 \\
+p_0(x_0) &= 1 + 2x_0 + 3x_0^2 \\ 
+p_0(2) &= 17 \\ 
+p_0(-2) &= 9 \\\\
+\text{Layer 2:} & \\
+x_1&=x_0^2 \\
+p_1(x_1) &= 3 + 3x_1 \\
+p_1(2^2) &= 15 \\ 
+p_1(-2^2) &= 9 \\\\
+\text{Layer 3:} & \\
+x_2&=x_1^2 \\
+p_2(x_2) &= 6 \\
+p_2((2^2)^2) &= 6 \\\\
+\end{align}
+$$
+
+
+
+In this manner, the prover can keep calculating for the whole domain. Then they would response the verifier query with the evaluations across layers for checking if an instance $a_i$ corresponding to the query is in the $\mathcal{L}\left(\mathcal{R}_{\text{consistent-layers }}\right)$
```

Comment:
> keep calculating for the whole domain

what do you mean by that?

also it's the first time you talk about the verifier doing queries here. I think the typical explanations of FRI (ethstark, thaler, etc.) are usually clearer because they explain the commit phase and then the query phase, but here it's not clear what the protocol to follow is

Reply:
I was trying to focus on describing how the evaluations can be done using the polynomials in different layers at the prover side, with the intention to avoid touching the commitment part for now.

Then I plan to explain on why it is necessary to incorporate the challenge by verifier and commit to these evaluations in another post or later section. I guess this arrangement might help separate the concepts between the "basic core calculations" and the security component.

Maybe I should explain this intention in this section to give a better clue on the role of this part in the protocol.